### PR TITLE
[EDITED]Edited the doc for rs (about us)

### PR DESCRIPTION
**Pull request by preyanskhasingh**
[EDITED] Edited the required sections of the code as reviewed according to issue no. #146

### CODE REVIEW

**Review by himanshu-03 — CHANGES_REQUESTED** [review]
Add images and change content as instructed!

**Comment by preyanskhasingh** [comment]
I have updated the documentation as required as of issue no. #146

File: docs/projects/resume-screener-website/frontend-components/about-us.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+title: About Us
+id: about-us
+description: About Us Component in Resume Screener Website
+sidebar_label: About Us
+keywords: [contact-us, staff]
+---
+
+The **About Us** page of the website will help you discover our story, mission, and values as an organization. It can be accessed through the navigation bar of the [**Resume Screener Website**](https://rs.tcetmumbai.in/).
+
+### Adding Star Resume Test Option
```
Comment (himanshu-03) [inline docs/projects/resume-screener-website/frontend-components/about-us.mdx:11]: That is a button not option

**Review by himanshu-03 — CHANGES_REQUESTED** [review]
Dont just cut short the code already, its hardly  20 lines of code

**Review by himanshu-03 — CHANGES_REQUESTED** [review]

**Comment by preyanskhasingh** [comment]
I have made the required changes on the code as of issue #146

**Review by himanshu-03 — APPROVED** [review]
LGTM!

**Pull request merged**